src/helpers/await.ts

```diff
@@ -1,17 +1,19 @@
 export type Awaitable<T> = T | PromiseLike<T>;
 
 export interface Signal {
-  signal?: AbortSignal;
+  signal: AbortSignal;
+  abort: () => void;
 }
 
 export interface Interval {
-  interval?: number;
+  interval: number;
+  cancel: () => void;
 }
 
-export type SignalAndInterval = Signal & Interval;
+export type PollingController = Signal & Interval;
```

Comment:
Changing this to a name that more closely describes what it does. I had to dig into the source code to figure this out when I first saw this.